### ISSUE TITLE
Missing OK button on information dialog boxes

### ISSUE THREAD

**Issue by cdrage** [containers/podman-desktop#2245]
### Bug description

When we were to do:
```
 extensionApi.window.showInformationMessage(`Docker Compose ${selectedRelease.label} installed`);
 ```
 
 It would automatically add an "OK" button for us to click.
 
 I've noticed this is missing in our most recent dialog changes.

![Screenshot 2023-04-26 at 11 03 02 AM](https://user-images.githubusercontent.com/6422176/234619393-fefbf166-7676-46d1-9573-d65eab21c622.png)

### Operating system

Mac M1

### Version

0.14.1

### Steps to reproduce

Go to install Compose or Kind, or use example code. You'll be missing an OK button for information dialog.

**Comment by deboer-tim** [comment]
According to the Electron documentation this should only have worked on Windows before so I missed it... was already creating an unrelated PR that would fix this though, will have a fix soon.

**Comment by deboer-tim** [comment]
Fixed via PR #2249.
